app/models/previousRegistrations/PreviousRegistrationDetails.scala

```diff
@@ -30,8 +30,8 @@ object PreviousRegistrationDetails {
 }
 
 case class PreviousSchemeDetails(
-                                  previousScheme: String,
-                                  previousSchemeNumbers: PreviousSchemeNumbers
+                                  previousScheme: Option[String],
+                                  previousSchemeNumbers: Option[PreviousSchemeNumbers]
```

Comment:
I think the preference would be not to adjust this as optional and to create another model to handle incomplete scenarios.
I think this model is used elsewhere and completion checks shouldn't impact that

Reply:
Changes reverted. Used existing model to handle incomplete scenario.